src/pages/styledcomponents/styledExerciseFormPage.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const StyledExercisePage = styled.div`
+	min-height: 100vh;
+	background-color: #acebaa;
+`;
```

Comment:
The name is misleading because the style applies to a  `div` is not a page  🤯
Please change.